src/components/HomePage/Category.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircleArrowRight } from '@fortawesome/free-solid-svg-icons';
+import { Link } from 'react-router-dom';
+import styles from './Category.module.css';
+import { actions } from '../../redux/covid/covid';
+import IMAGES from '../../images/images';
+import loader from '../../images/1492.gif';
+
+function CovidCase() {
+  const dispatch = useDispatch();
+  const loading = useSelector((state) => state.covidCases.loading);
+  const handleCovidClick = (name) => {
+    dispatch(actions.filterCases(name));
+  };
```

Comment:
- [ ] On the homepage you are yet to implement the filtering (or search) functionality as requested by the previous reviewer. Kindly implement that as well.

  ![search](https://user-images.githubusercontent.com/54848363/194660515-5c976e27-052d-437e-9a02-6059fab1c540.jpg)